src/runtime_lib/infra_gapbs/intersections.h

```diff
@@ -0,0 +1,336 @@
+// Copyright (c) 2015, The Regents of the University of California (Regents)
+// See LICENSE.txt for license details
+
+#include <algorithm>
+#include <cinttypes>
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include "benchmark.h"
+#include "builder.h"
+#include "command_line.h"
+#include "graph.h"
+#include "pvector.h"
+#include "bitmap.h"
+#include "timer.h"
+
+
+using namespace std;
+
+
+/*
+Runtime library for various set intersection methods
+*/
+
+
+
+size_t MAX_LIMIT = 61578415;
+
+
+//TODO: there might be some issues with the allocating space (for now I am just allocating big enough space)
+size_t intersectSortedNodeSetBitset(NodeID* A, NodeID* B, size_t totalA, size_t totalB) {
+  
+  size_t total = 0;
+
+  //we only want to build bitmap for bigger array
+  if (totalA > totalB) {
+
+    Bitmap a_array(MAX_LIMIT);
+    a_array.reset();
+
+    for(size_t i = 0; i < totalA; i++){
+        a_array.set_bit(*(A+i));
+    }
+
+    for(size_t j = 0; j < totalB; j++){
+        if (a_array.get_bit(*(B+j))){
+            total++;
+        }
+    }
+
+  } 
+  else {
+    Bitmap b_array(MAX_LIMIT);
+    b_array.reset();
+
+    for(size_t i = 0; i < totalB; i++){
+        b_array.set_bit(*(B+i));
+    }
+
+    for(size_t j = 0; j < totalA; j++){
+        if (b_array.get_bit(*(A+j))){
+            total++;
+        }
+    }
+  }
+
+  
```

Comment:
remove the extra spaces. Actually, just use Clion to do a reformat of this file, it should clean up all the extra spaces nicely.